src/applications/pact-act/containers/questions/burn-pit/BurnPit-2-1-1.jsx

```diff
@@ -78,11 +81,11 @@ const mapDispatchToProps = {
 
 BurnPit211.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setBurnPit211: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.